dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -780,7 +780,7 @@ public Workflow publish(@ApiParam(hidden = true) @Parameter(hidden = true, name
     @ApiOperation(value = "List all published workflows.", tags = {
         "workflows"}, notes = "NO authentication", response = Workflow.class, responseContainer = "List")
     public List<Workflow> allPublishedWorkflows(
-        @ApiParam(value = "Start index of paging. Pagination results can be based on numbers or other values chosen by the registry implementor (for example, SHA values). If this exceeds the current result set return an empty set.  If not specified in the request, this will start at the beginning of the results.") @QueryParam("offset") String offset,
```

Comment:
@denis-yuen I also changed this endpoint because it used `entryDAO.findAllPublished`. If you'd prefer I can try to isolate the change to only the `/containers/published` endpoint.

Reply:
that seems fine, we're generally the only user of these non-standardized endpoints (i.e. not GA4GH TRS)
that said, you should probably submit a paired PR linked to this one using these new swagger/openapi definitions in the UI due to the type change

Reply:
I'm confused about submitting a PR linked to this one in the UI repo.
I've tried to generate new definitions by using `npm run prebuild`
https://github.com/dockstore/dockstore-ui2/blob/77679781a26213565cca03b4f729c2adb1d6f07a/package.json#L16
I used the following patch to get the command above to read the openapi.yaml files from this PR.

```
Index: package.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/package.json b/package.json
--- a/package.json	(revision 77679781a26213565cca03b4f729c2adb1d6f07a)
+++ b/package.json	(date 1652133400670)
@@ -4,7 +4,7 @@
   "license": "Apache License 2.0",
   "config": {
     "webservice_version": "1.13.0-alpha.6",
-    "use_circle": true,
+    "use_circle": false,
     "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/7557/workflows/432342bd-9a3e-4648-88a3-5b77de6b8879/jobs/18860/artifacts",
     "circle_build_id": "18860",
     "base_branch": "develop"
Index: scripts/generate-openapi-script.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/scripts/generate-openapi-script.sh b/scripts/generate-openapi-script.sh
--- a/scripts/generate-openapi-script.sh	(revision 77679781a26213565cca03b4f729c2adb1d6f07a)
+++ b/scripts/generate-openapi-script.sh	(date 1652132127735)
@@ -8,7 +8,7 @@
 BASE_PATH="https://raw.githubusercontent.com/dockstore/dockstore/$npm_package_config_webservice_version"
 # DOCKSTORE-2428 - demo how to add new workflow language, generate from local copy of swagger
 # Uncomment this to use your local copy of swagger instead
-# BASE_PATH="../dockstore"
+BASE_PATH="../dockstore"
 
 
 wget --no-verbose https://repo.maven.apache.org/maven2/org/openapitools/openapi-generator-cli/${GENERATOR_VERSION}/openapi-generator-cli-${GENERATOR_VERSION}.jar -O openapi-generator-cli.jar
```

But there was not a diff that was generated, is there another command that I need to run? Or a CI test fixture that needs to be updated?